flang/include/flang/Runtime/command.h

```diff
@@ -11,6 +11,13 @@
 
 #include "flang/Runtime/entry-names.h"
 
+#ifdef _WIN32
+// On Windows* OS GetCurrentProcessId returns int (not pid_t)
+typedef int64 pid_t;
```

Comment:
`int64` isn't a valid type, you need `std::int64_t`. To bring that type into scope you'll need to move the `#include <cstdint>` from below to above this '#ifdef` but that shouldn't be an issue.

Reply:
Sorry, I just checked and the return type of GetCurrentProcessID is actually `DWORD` so it should be
`typedef DWORD pid_t;`
I think I got this wrong when I suggested the change so sorry about that!

Reply:
Thanks a lot, a new patch has been updated, shoud pass ci now